openrr-planner/src/collision/collision_detector.rs

```diff
@@ -452,13 +452,21 @@ mod tests {
 
 /// Convert urdf object into openrr_planner/ncollide3d object
 pub trait FromUrdf {
-    fn from_urdf_robot(robot: &urdf_rs::Robot) -> Self;
-    fn from_urdf_file<P>(path: P) -> ::std::result::Result<Self, urdf_rs::UrdfError>
+    fn from_urdf_robot_with_base_dir(robot: &urdf_rs::Robot, path: Option<&Path>) -> Self;
+    fn from_urdf_robot(robot: &urdf_rs::Robot) -> Self
```

Comment:
If there is no longer any code that actually calls this API, in other words, if we should always call from_urdf_robot_with_base_dir, we may want to change from_urdf_robot to take `base_path: Option<&Path>` instead of adding from_urdf_robot_with_base_dir.

Reply:
It's following the same convention as the other existing `xxx_with_base_dir`